bazaarvoice/emodb
# add postman tests for get uac/1/role/{group} endpoint

## What Are We Doing Here?

Adding new postman tests for get /uac/1/role/{group}

## How to Test and Verify

1. Check out this PR
2. Run This PR on Jenkins with pipeline stage to execute these tests
3. Verify that tests for endpoint are run and passed

## Risk
broken tests
unstable tests
not executable tests

### Level 

`Low`

### Required Testing

`Manual`
### Risk Summary

Adding tests that are unstable or  testing system in a wrong manner

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
